app/src/js/actions/index.js

```diff
@@ -781,8 +781,6 @@ export const deleteRule = (ruleName) => ({
 
 export const enableRule = (payload) => {
   const rule = clonedeep(payload);
-  delete rule['queriedAt'];
-  delete rule['timestamp'];
```

Comment:
Updating this action to get a rule using the individual rule GET endpoint means that an extraneous `timestamp` property is not returned, so we don't have to delete it

Reply:
Yup makes sense! Thanks for the explanation!